src/base/DeltaResolver.sol

```diff
@@ -76,7 +76,7 @@ abstract contract DeltaResolver is ImmutableState {
     }
 
     /// @notice Calculates the amount for a settle action
-    function _mapSettleAmount(uint256 amount, Currency currency) internal view returns (uint256) {
+    function _mapSettleAmount(uint256 amount, Currency currency) internal view virtual returns (uint256) {
```

Comment:
We should use tags to make sure that we are keeping track of what DeltaResolver contracts are used for which protocols

Ie with this change we will have now a few different versions of DeltaResolver

Reply:
It realistically should not affect the bytecode for any deployed contracts